src/common/net.cc

```diff
@@ -238,9 +238,7 @@ Address::init(const std::string& addr) {
         family_ = AF_INET6;
         try {
             in6_addr addr6;
-            char buff6[INET6_ADDRSTRLEN + 1] = {0, };
-            std::copy(host_.begin(), host_.end(), buff6);
-            inet_pton(AF_INET6, buff6, &(addr6.s6_addr16));
+            inet_pton(AF_INET6, host_.c_str(), &(addr6.s6_addr16));
```

Comment:
Please, merge net.cc with master

Reply:
merged - we now do:
inet_pton(AF_INET6, host_.c_str(), &(addr6.s6_addr16));

i.e: just pass the 'host_' to inet_pton. 
I don't know why a copy was done before that (just wasted cycles + valgrind issue)